Always execute subpackage split functions in $pkgdir. This allows amove
arguments to be safely globbed before calling.

--- a/abuild.in
+++ b/abuild.in
@@ -933,6 +933,8 @@ pre_split() {
 	# the subpackages should not inherit these from main package
 	provides=""
 	install_if=""
+
+	cd "$pkgdir"
 }
 
 prepare_subpackages() {
